paddown.py

```diff
@@ -44,7 +45,9 @@ def decrypt_block(self, c_i):
 
         c_previous = bytearray(b"\x00" * self.blocksize)
         intermediate = bytearray(b"\x00" * self.blocksize)
+
         for i in range(self.blocksize):
+            self.progress_bar(i, self.blocksize, "Decrypting")
```

Comment:
This placement could be the issue with the progress bar not reaching 100%

Reply:
The problem found to be in my method logic. As a sample output:
```bash
[#########################################################] 100%  Decrypting 

bytearray(b'This is a padded plaintext\x06\x06\x06\x06\x06\x06')
```
I added a new line print statement in the `decrypt` method to let it be on a new line from the percentage count.